test/development/app-dir/dynamic-io-dev-errors/dynamic-io-dev-errors.test.ts

```diff
@@ -75,7 +75,9 @@ describe('Dynamic IO Dev Errors', () => {
           ? '\n    at main (<anonymous>)' +
             '\n    at body (<anonymous>)' +
             '\n    at html (<anonymous>)' +
-            '\n    at Root [Server] (<anonymous>)'
+            '\n    at Root [Server] (<anonymous>)' +
+            // Just need some string to assert that this is the whole stack
+            '\n GET /no-accessed-data 200'
```

Comment:
Now lists more internals frames now that `ignoreList` is no longer filled in `next-server` bundles.